### ISSUE TITLE
What reports do you find most valuable?

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#161]
Different users need different reporting, and as we build #127 we need to think about how to make reports as useful as possible.

For instance...
- a dev might want lighthouse descriptions, code snippets, and links to WCAG standards.
- an accessibility manager might want teaching tools like @joelhsmith [videos](https://www.youtube.com/playlist?list=PLF4_5fOOz5tlYmG6BnX8q_ILEzBPNrDbK), [deque university](https://dequeuniversity.com/) links, and screenshots of error areas.

I know there are existing [checkpoints](https://www.w3.org/WAI/EO/wiki/Accessibility_Checkpoint_Master_List) and [roles](https://www.w3.org/WAI/EO/wiki/ARRM_Project_-_Accessibility_Roles_and_Responsibilities_Mapping), defined by W3. Do those roles speak to how you use accessibility?

Equalify reports should be dynamic and useful. Please post any examples you have! Thanks!

**Comment by syzygy333** [comment]
Came here to say "Lighthouse," but I'd like to change my answer to "all of the above" (minus videos).

**Comment by bbertucc** [comment]
Closing this question for lack of activity, but feel free to open the issue with more examples of what reports are important.
